xml/docu_styleguide.terminology.xml

```diff
@@ -907,10 +912,9 @@
       <entry>noun; spelling according to project standard</entry>
      </row>
      <row>
-      <entry>list box</entry>
-      <entry>list, list field</entry>
-      <entry>noun; GUI element that is a list showing multiple elements even
-              before interacting with it
+      <entry>list</entry>
+      <entry>list box, list field</entry>
```

Comment:
I didn't dare to forbid <list box> in TermWeb, I used 'Accepted' status for this